web_src/js/components/RepoActionView.vue

```diff
@@ -834,6 +813,22 @@ export function initRepositoryActionView() {
   margin-left: 10px;
 }
 
+.full.height > .action-view-right {
```

Comment:
The comments like `/* selectors here are intentionally exact to only match fullscreen */` should also be moved here?

Reply:
done